rero_ils/modules/notifications/api.py

```diff
@@ -207,6 +202,19 @@ def init_loan(self):
             self.loan = Loan.get_record_by_pid(self.loan_pid)
         return self.loan
 
+    def _get_library_informations(self, library_pid):
+        """."""
+        library = Library.get_record_by_pid(library_pid)
+        data = dict()
+        for field in [
+            'pid', 'email', 'name', 'address', 'notification_settings',
+            'communication_language', 'next_open'
```

Comment:
`next_open` isn't a field, it's a function. So you couldn't use `library.get('next_open')`.
I think you need to keep original code 
```
keep_until = datetime.now(timezone.utc) + timedelta(days=10)
next_open = library.next_open(keep_until)
```

Reply:
Thanks for reviewing!